.github/workflows/unit-tests.yaml

```
@@ -43,21 +43,11 @@ jobs:
         run: "npm install -g @go-task/cli"
 
       - if: "'macos-14' == matrix.os"
-        name: "Install macOS 14 deps: coreutils (for md5sum) and Apple Clang 16 (for C++20)"
+        name: "Install macOS 14 deps: coreutils (for md5sum)"
```

Since this line only runs when run on macOS, I feel like we can simplify the name.

```suggestion
        name: "Install coreutils (for md5sum)"
```